src/pages/Lobby.tsx

```diff
@@ -1,8 +1,7 @@
 import styled from 'styled-components';
-
-// components
 import App from '../apps/pillarx-app';
 
+// components
```

Comment:
That must be an auto-formatting situation, but this needs to go back where it was :)

Reply:
Oops! Changed 🙏